index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navmenu">
+      <h3>My Logo</h3>
+    </nav>
+    <img src="image/Union.png" alt="" />
+
+  </header>
+  <section class="backgroundimg">
+    <div class="menuSection">
+      <h1 class="introduction">I’m Mariane Glad to see you!</h1>
+      <p class="Whoiam">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+    </div>
+    <div class="menuSection">
+      <a href="#" target="_blank">LET'S CONNECT</a>
+      <ul class="socialMedia">
+        <li>
+          <img src="image/Twitter.png" alt="" />
+
+        </li>
+        <li>
+          <img src="image/Linkedin.png" alt="" />
+
+        </li>
+        <li>
+          <img src="image/Meduim.png" alt="" />
+
+        </li>
+        <li>
+          <img src="image/Github.png" alt="" />
+
+        </li>
+        <li>
+          <img src="image/angellist.png" alt="" />
+
+        </li>
+      </ul>
+    </div>
+  </section>
+  <div id="cards">
+
+    <div id="flex">
+      <div class="card">
+        <img class="img" src="image/card-1.svg" alt="">
+        <div class="content">
+          <div class="topic">Tonic</div>
```

Comment:
- [ ] I think you should consider wrapping the project title (e.g."Tonic") in a header tag hx(h1-h6). This is essential for your website accessibility and also for screen readers to be able to use and understand the project 👍 .
e.g          ` <h2 class="topic">Tonic</h2>`